localscripts/generate-po2json.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+[ -x "$(command -v npm)" ] || { >&2 echo "npm is required but not installed, exiting."; exit 1; }
+
+echo ""
+echo "Installing po2json..."
+echo ""
+npm install po2json
+
+echo ""
+echo "Generating .po to .json files..."
+echo ""
+
+pofilenames=$(ls ./translations/planet4-plugin-gutenberg-blocks/languages/*.po)
+for pofilename in $pofilenames
+do
+   backend_domain='planet4-blocks-backend'
```

Comment:
Sorry I didn't correct my earlier comment, but that suggestion (to use `backend_domain` as the name) was before I made the comment to use the suffix. Now that we use the suffix this name can be just `domain`, or maybe `text_domain`.